app/processing/schedule/get-scheduled-settlements.js

```diff
@@ -10,19 +9,19 @@ const getScheduledSettlements = async (started, transaction) => {
     skipLocked: true,
     limit: config.scheduleProcessingMaxBatchSize,
     transaction,
-    attributes: [
-      'scheduleId',
-      'settlementId'
-    ],
     include: [{
       model: db.settlement,
       as: 'settlements',
       attributes: []
     }],
+    attributes: [
+      'scheduleId',
+      'settlementId'
+    ],
     where: {
       category: STATEMENT,
       completed: null,
-      '$settlements.sourceSystem$': SOURCE_SYSTEM.SFI,
```

Comment:
This line of code is removed as statement generation is not limited to SFI scheme but any scheme that has been set as active and current in the database.

Reply:
Thanks for the context